src/components/TasksTable/index.jsx

```diff
@@ -14,6 +14,7 @@ import { camelCase } from 'change-case';
 import { formatDistance } from 'date-fns';
 import MenuItem from '@material-ui/core/MenuItem';
 import TextField from '@material-ui/core/TextField';
+import Tooltip from '@material-ui/core/Tooltip';
```

Comment:
We can simply use the title attribute for a basic tooltip. https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/title.